### PR TITLE
Add Coverage Badge

### PR DESCRIPTION
**Pull request by DrFaust92**
* Remove david-dm badge
* Add codecov - badge + integration

### CODE REVIEW

**Comment by DrFaust92** [comment]
Resolves #35

**Comment by DrFaust92** [comment]
I had trouble so there are many junk commits, needs to be squashed.

**Comment by lirantal** [comment]
No worries. Looking good! Thanks a bunch 🙏❤️

Seems you forgot to add the codecov badge to the README? :-)

File: README.md [hunk 1]
```diff
@@ -5,7 +5,6 @@
 [![view on npm](http://img.shields.io/npm/v/express-version-route.svg)](https://www.npmjs.org/package/express-version-route)
 [![view on npm](http://img.shields.io/npm/l/express-version-route.svg)](https://www.npmjs.org/package/express-version-route)
 [![npm module downloads](http://img.shields.io/npm/dt/express-version-route.svg)](https://www.npmjs.org/package/express-version-route)
-[![Dependency Status](https://david-dm.org/lirantal/express-version-route.svg)](https://david-dm.org/lirantal/express-version-route)
```
Comment (lirantal) [inline README.md:8]: Missed adding the codecov badge :)

**Review by lirantal — CHANGES_REQUESTED** [review]

**Comment by lirantal** [comment]
Thanks @DrFaust92 👌

**Pull request merged**